source/ui/beamline/AMExtendedControlEditor.cpp

```diff
@@ -456,6 +462,20 @@ QSize AMExtendedControlEditor::sizeHint() const{
 	return newHint;
 }
 
+void AMExtendedControlEditor::onControlMinimumValueChanged()
+{
+	if (control_) {
```

Comment:
No, I'm thinking that since controls won't, in general, be emitting the min/max changed singals I'm worried that the default behaviour/values will cause issues in the editor.  Such as only allowing -1 or 0 or some such.